shared/dev/dumb-sheet.render.native.js

```diff
@@ -40,32 +37,33 @@ class Render extends Component<void, Props, any> {
     }, 0)
   }
 
-  _continueTest () {
-    if (isTesting && this.state.testIndex !== -1) {
+  _increment () {
+    if (this.props.autoIncrement && this.state.testIndex !== -1) {
       const {componentsOnly} = this._getComponents()
       if (this.state.testIndex >= componentsOnly.length) {
         this.setState({testIndex: -1})
       } else {
-        this.setState({testIndex: this.state.testIndex + TEST_RENDER_NUMBER})
+        this.setState({testIndex: this.state.testIndex + 1})
       }
     }
   }
 
   componentDidMount () {
-    if (isTesting) {
-      this._continueTest()
+    if (this.props.autoIncrement) {
+      this._increment()
     }
   }
 
   componentDidUpdate () {
-    setTimeout(() => {
-      this._continueTest()
-    }, 1)
+    if (this.props.autoIncrement) {
+      setTimeout(() => {
+        this._increment()
+      }, 1)
```

Comment:
also the setTimeout 1 was cause you can't cal setState in this function actually, it has to be deferred